Automattic/jetpack
# Add Advanced Plan to Social pricing table

This change adds the Advanced plan to appear in Jetpack Social's pricing table.
It also contains some UI changes to match the current design.

## Proposed changes:

* Added a new column with the Advanced plan
* Moved the Intro offer disclaimer to the bottom above the ToS via a prop
* Changed the naming of the first feature to say `in 30 days` instead of `per month` to avoid confusion (p1672914742301729/1671124809.949759-slack-C02JJ910CNL).
* Added `Coming soon` to features that we do not have yet. p1673869169329449/1673868332.570029-slack-C02JJ910CNL
* Added tooltips to features based on this: p1HpG7-jyl-p2#comment-59815

### Other information:


## Jetpack product discussion



Design P2: p1HpG7-jD8-p2
Figma file: 9TYJIUjkAEKweGyuQ8zqm6-fi-2915%3A7197

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Open the design P2 and Figma file.
* To see the pricing table on your site just change `showPricingPage` to true in `projects/plugins/social/src/js/components/admin-page/index.jsx`
* Compare pricing table with design.
